public/media_engagement/auth.html

```diff
@@ -11,7 +11,7 @@
     <script>
         // FirebaseUI config.
         var uiConfig = {
-            signInSuccessUrl: "media_engagement.html",
+            signInSuccessUrl: "imaqal_sms_traffic.html",
```

Comment:
I think it would be better to redirect back to `../coda/coding_progress.html` because the SMS traffic pages are not permanent.

Reply:
@DanielMwendwa  whats the progress on this?

Reply:
finished